docs: nextval(seq) as a column default value (#1737)
databendlabs/databend-docs: docs/en/sql-reference/10-sql-commands/00-ddl/01-table/10-ddl-create-table.md

By default, **all columns are nullable(NULL)** in Databend. If you need a column that does not allow NULL values, use the NOT NULL constraint. For more information, see [NULL Values and NOT NULL Constraint](../../../00-sql-reference/10-data-types/index.md).

## Column Default Values

`DEFAULT <expr>` sets a default value for the column when no explicit expression is provided. The default expression can be:

- A fixed constant, such as `Marketing` for the `department` column in the example below.

> -  An expression with no input arguments and returns a scalar value, such as `1 + 1`, `NOW()` or `UUID()`.

Reviewer: ```suggestion
- An expression with no input arguments and returns a scalar value, such as `1 + 1`, `NOW()` or `UUID()`.
```

- A dynamically generated value from a sequence, such as `NEXTVAL(staff_id_seq)` for the `staff_id` column in the example below.
  - NEXTVAL must be used as a standalone default value; expressions like `NEXTVAL(seq1) + 1` are not supported.

>   - When using COPY INTO without explicitly specifying a column with a DEFAULT NEXTVAL value, Databend will automatically generate values for that column.

Reviewer: ```suggestion
```

By default, **all columns are nullable(NULL)** in Databend. If you need a column that does not allow NULL values, use the NOT NULL constraint. For more information, see [NULL Values and NOT NULL Constraint](../../../00-sql-reference/10-data-types/index.md).

## Column Default Values

> `DEFAULT <expr>` sets a default value for the column when no explicit value is provided. The default value can be:

Reviewer: ```suggestion
`DEFAULT <expr>` sets a default value for the column when no explicit expression is provided. The default expression can be:
```

- A fixed constant, such as `Marketing` for the `department` column in the example below.

> - The result of an expression with no input arguments, such as `1 + 1`.

Reviewer: ```suggestion
-  An expression with no input arguments and returns a scalar value, such as `1 + 1`, `NOW()` or `UUID()`.
```

> - The result of a non-deterministic function, such as `NOW()` or `UUID()`.

Reviewer: ```suggestion
```
